waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -292,10 +292,10 @@ genEnvValidationScript =
 genWebSockets :: AppSpec -> Generator [FileDraft]
 genWebSockets spec
   | AS.WS.areWebSocketsUsed spec =
-    sequence
-      [ genFileCopy [relfile|webSocket.ts|],
-        genWebSocketProvider spec
-      ]
+      sequence
```

Comment:
Ormolu formatted this for me. I believe that's something that Ormolu complained about in the CI anyways.